src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -38,6 +40,9 @@ public class WebScriptHostManager : ScriptHostManager
         private readonly ScriptHostConfiguration _config;
         private readonly ISwaggerDocumentManager _swaggerDocumentManager;
         private readonly object _syncLock = new object();
+
+        private readonly WebJobsSdkExtensionHookProvider _bindingWebhookProvider = new WebJobsSdkExtensionHookProvider();
```

Comment:
IWebHookProvider  exposes the registration half (that extensions call); this class needs to reader half.